stubs/pywin32/isapi/threaded_extension.pyi

```diff
@@ -1,26 +1,28 @@
 import threading
 from _typeshed import Incomplete
+from collections.abc import Callable
+from typing import Final, Literal
 
 import isapi.simple
 
-ISAPI_REQUEST: int
-ISAPI_SHUTDOWN: int
+ISAPI_REQUEST: Final = 1
+ISAPI_SHUTDOWN: Final = 2
 
 class WorkerThread(threading.Thread):
     running: bool
-    io_req_port: Incomplete
+    io_req_port: int
     extension: Incomplete
-    def __init__(self, extension, io_req_port) -> None: ...
+    def __init__(self, extension, io_req_port: int) -> None: ...
```

Comment:
This extension needs to have a `dispatch_map` in `WorkerThread.run`, so it has to be a `ThreadPoolExtension`
```suggestion
    def __init__(self, extension: ThreadPoolExtension, io_req_port: int) -> None: ...
```